requirements-arm.txt

```diff
@@ -0,0 +1,8 @@
+# Common dependencies
+-r requirements-common.txt
+
+torch == 2.4.0
+torchvision  # required for the image processor of phi3v, this must be updated alongside torch
```

Comment:
Honestly I haven't checked through the versions so that I don't know which is supported or not. This torch version is just copied from the x86 CPU backend. This could be future work.